src/builders/utils/uploader.ts

```diff
@@ -29,3 +39,62 @@ export async function uploadBuildToS3(ctx: IUploadCtx) {
     return fileLocation;
   }
 }
+
+export async function uploadBuildToTestFlight(ctx: IContext, options: IJobOptions, bundleIdentifier?: string) {
+  const fastlaneEnvVars = {
+    FASTLANE_USER: options.appleId,
+    FASTLANE_SKIP_UPDATE_CHECK: 1,
+    FASTLANE_DISABLE_COLORS: 1,
+    CI: 1,
+    LC_ALL: 'en_US.UTF-8',
+    FASTLANE_PASSWORD: options.applePassword,
+    FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD: options.appSpecificPassword,
```

Comment:
Does it make sense to pass both `FASTLANE_PASSWORD` and `FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD` as env vars? If not then we need to add some condition here.